js/tests/index.html

```diff
@@ -5,6 +5,8 @@
     <title>Bootstrap Plugin Test Suite</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
+    <link rel="stylesheet" href="../../dist/css/bootstrap.css">
```

Comment:
Not a big fan of adding the entire bootstrap css here, I prefer to cherry-pick the CSS rules we need and add it when we need it (and remove them after)